hw/ip/edn/dv/edn_sim_cfg.hjson

```diff
@@ -12,7 +12,7 @@
   tb: tb
 
   // Simulator used to sign off this block
-  tool: xcelium
+  tool: vcs
```

Comment:
@senelson7 we can use run xcelium with `-t xcelium` in dvsim. I just update there so it is easier for me to look at coverage waivers.